packages/neuron-wallet/package.json

```diff
@@ -20,7 +20,8 @@
     "test": "jest --color",
     "lint": "eslint 'src/**/*.{js,ts}'",
     "package": "./script/package.sh",
-    "postinstall": "npm rebuild tiny-secp256k1"
+    "postinstall": "npm rebuild tiny-secp256k1",
+    "testStore": "ava"
```

Comment:
We don't have any script with `camelCase` names.

Reply:
ok